cmd/oceanbench/broadcast.go

```diff
@@ -145,8 +145,10 @@ type BroadcastConfig struct {
 	StartFailures     int           // The number of times the broadcast has failed to start.
 	Transitioning     bool          // If the broadcast is transition from live to slate, or vice versa.
 	StateData         []byte        // States will be marshalled and their data stored here.
+	HardwareStateData []byte        // Hardware states will be marshalled and their data stored here.
 	Account           string        // The YouTube account email that this broadcast is associated with.
 	InFailure         bool          // True if the broadcast is in a failure state.
+	TimeCreated       time.Time     // The date and time the broadcast was created.
```

Comment:
Couldn't we just use the scheduled start time from the broadcast object data ?

Reply:
Do you mean the Start field of the BroadcastConfig struct? It's my understanding that that doesn't get updated to reflect the date of the stream, we just use the time portion of that field for start and stop times.

Are you suggesting that we update the date in the Start field to be current when the broadcast is created, then just use that.. That could work.

Reply:
I mean more like the youtube broadcast object.

Reply:
Done.